iron-request.html

```diff
@@ -425,7 +425,9 @@
     _wwwFormUrlEncodePiece: function(str) {
       // Spec says to normalize newlines to \r\n and replace %20 spaces with +.
       // jQuery does this as well, so this is likely to be widely compatible.
-      if (str === null) {
+      try {
+        str.toString
```

Comment:
This issue has come up again because this did not cover enough cases where `str.toString` was throwing an exception. Some values like NaN, -0, 0, '0', and '-0' do end up having `toString` functions which is why I strayed away from falsey values.